cost/azure/unused_volumes/azure_unused_volumes_meta_parent.pt

```diff
@@ -0,0 +1,923 @@
+name "Azure Unused Volumes Meta Parent"
+rs_pt_ver 20180301
+type "policy"
+short_description "Applies and manages \"child\" [Azure Unused Volumes](https://github.com/flexera-public/policy_templates/tree/master/cost/azure/unused_volumes) Policies."
+long_description ""
+severity "low"
+category "Cost"
+default_frequency "daily"
+info(
+  version: "0.1",
+  provider: "Azure",
+  service: "Storage",
+  policy_set: "Unused Volumes",
+  recommendation_type: "Usage Reduction",
```

Comment:
These fields should not be in the info block of the parent policy. If they are, then both the consolidated incident raised by the parent policy, and the individual incidents raised by the child policy, will be scraped, resulting in double recommendations in the Optimization dashboard.

I would recommend removing the policy_set and recommendation_type fields from here.

Reply:
Removed, thank you!